CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 3.4.0 (2019-09-24)
+
+- Added the single file attachment component. [#60](https://github.com/blackbaud/skyux-forms/pull/60)
+
+- Added the character count component. [#61](https://github.com/blackbaud/skyux-forms/pull/61)
+
+- Fixed the file drop component trash icon. [#62](https://github.com/blackbaud/skyux-forms/pull/62)
+
+- Added CONTRIBUTING.md file. [#64](https://github.com/blackbaud/skyux-forms/pull/64)
```

Comment:
The `Added` portions should be grouped together. Could we move this up with the others?